Sources/Apodini/Request/RequestInjectable.swift

```diff
@@ -12,12 +12,30 @@ import Runtime
 
 protocol RequestInjectable {
     mutating func inject(using request: Vapor.Request, with decoder: RequestInjectableDecoder?) throws
+    func visit(_ visitor: RequestInjectableVisitor)
```

Comment:
I know this naming `visit(visitor: ...)` is similar to the `SyntaxTreeVisitor` but I got a little confused why we do not use the more common `accept(visitor: ...)` method. Is it for a certain reason? Just from the patterns lecture and wikipedia, I would assume, that the `Visitor` calls `visit()` whereas the `Visitable` calls `accept()`.

Reply:
Yeah, I pretty much adopted the naming from the `SyntaxTreeVisitor`. 😅 But yes you are correct. We may want to change that to be consistent with the literature. I got a issue open right know for the SyntaxTreeVisitor to fix a spelling mistake in the class name, we may just incorporate that naming change there and adjust this PR as well?

Reply:
solved via e7383a3

Reply:
for the sake of not including two different things in one PR I already merged #54 and we may just create an additional PR to change the naming there as well.